docs/aws/README.md

```diff
@@ -1,11 +1,31 @@
-# AWS
+# Getting Started with Karpenter on AWS
```

Comment:
I might just call this `Getting Started`, as it's already scoped to the AWS page.

Reply:
I'm planning on having this page linked prominently from the homepage.